src/sagemaker/feature_store/dataset_builder.py

```diff
@@ -19,14 +19,27 @@
 import datetime
 from enum import Enum
 import os
-from typing import Any, Dict, List, Tuple, Union
+from typing import Any, Dict, List, Tuple, Union, TYPE_CHECKING
 
+import logging
 import attr
-import pandas as pd
 
 from sagemaker import Session, s3, utils
 from sagemaker.feature_store.feature_group import FeatureDefinition, FeatureGroup, FeatureTypeEnum
 
+from sagemaker.utils import DeferredError, is_likely_a_pandas_df
+
+logger = logging.getLogger(__name__)
+
+try:
+    import pandas as pd
+except ImportError as e:
+    logger.warning("pandas failed to import. Feature store features will be impaired or broken.")
```

Comment:
Would being explicit help in this case?

```suggestion
    logger.warning("pandas is not installed in your environment. Feature store features will be impaired or broken.")
```

This suggestion also applies to other warning messages.

Reply:
I want to keep it consistent with other messages of the SDK and also this message reflects correctly the import issue

Reply:
Can we also add an instruction to install from extras, would it help for users to unblock when hit with this issue.